http_logs/track.py

```diff
@@ -7,10 +13,36 @@ async def reindex_async(es, params):
     result = await es.reindex(body=params.get("body"), request_timeout=params.get("request_timeout"))
     return result["total"], "docs"
 
+class RuntimeFieldResolver(loader.TrackProcessor):
+    PATTERN = re.compile('.+-from-(.+)-using-(.+)')
+
+    def on_after_load_track(self, t):
```

Comment:
One other thing to keep in mind is that both the `on_after_load_track` and `on_prepare_track` added on elastic/rally#1135 and available on after Rally 2.0.3 (released Jan 13 2021). So users using a slightly older version of Rally might have issues; I quickly tested by merging your PR to my fork / master branch and using Rally 2.0.2 things like `esrally info --track=http_logs` or event `esrally list tracks` worked. So I feel we should be ok if we keep the changes brought here only in the master branch of rally-tracks. Otherwise people using say, `http_logs/append-no-conflicts` against Elasticsearch 7.11 with Rally <2.0.3 will have problems.